django/api/models/rebate_historical.py

```diff
@@ -0,0 +1,28 @@
+from shortuuid.django_fields import ShortUUIDField
+from django.conf import settings
+from django.db.models import (
+    CharField,
+    IntegerField,
+    DateField,
+    BooleanField,
+    PROTECT,
+    ForeignKey,
+)
+
+from django.core.validators import MinLengthValidator
+
+from django_extensions.db.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+from ..models import GoElectricRebateApplication
+
+
+class rebate_historical(TimeStampedModel):
+    id = ShortUUIDField(length=16, primary_key=True, editable=False)

```

Comment:
We don't need a short uuid here for the rebates, we could prob use a regular auto incrementing id here.